src/briefcase/__main__.py

```diff
@@ -16,7 +17,7 @@ def main():
         command = Command(logger=logger, console=console)
         options = command.parse_options(extra=extra_cmdline)
         command.check_obsolete_data_dir()
-        command.parse_config("pyproject.toml")
+        command.parse_config(Path.cwd() / "pyproject.toml")
```

Comment:
This makes the path absolute, which means it can be tested reliably by monkeypatching cwd.